### PR TITLE
The second JLESC Workshop.

### PR DESCRIPTION
**Pull request by isabelheisters**
Signed-off-by: Isabel Heisters i.heisters@fz-juelich.de

### CODE REVIEW

File: _events_past/2014-11-24-2nd-jlesc-workshop.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: page_event
+title: 2nd JLESC Workshop
+date: 2016-02-19
```
Comment (torbjoernk) [inline _events_past/2014-11-24-2nd-jlesc-workshop.md:4]: Next time, try to correct this one, please :wink:

Comment (pancetta) [inline _events_past/2014-11-24-2nd-jlesc-workshop.md:4]: I don't get it.. what is wrong here?

Comment (torbjoernk) [inline _events_past/2014-11-24-2nd-jlesc-workshop.md:4]: February 19th isn't today. :wink: Yes, I'm nitpicking. Again. :wink: 

> _date_ is the date the page/article was created

Comment (pancetta) [inline _events_past/2014-11-24-2nd-jlesc-workshop.md:4]: Yeah, you are..

**Pull request merged**